tests/lib/eZ/RichText/Converter/Xslt/_fixtures/xhtml5/edit/lossy/008-anchor.xhtml5.edit.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<section xmlns="http://ez.no/namespaces/ezpublish5/xhtml5/edit">
+    <p>
+        <a name="top"/>some anchor with a name attribute.
```

Comment:
I thought the point here was to allow and map both `xml:id` to `id` and `xml:name` to `name`. The tests however suggest that id is swapped with name, which TBH doesn't make sense.

Reply:
The `name` attribute isn't used for anchors anywhere, that's why I decided to just map it to the existing one which is `id`.

Reply:
Actually there is not, fair point. 😅 I will change that.

Reply:
@alongosz I checked [doc reference](https://docbook.org/schemas/sdocbook/elements/anchor.html) and tested (validation fails) mapping to `name` attribute and it turns out that it stands against schema for anchors most likely. Should we aim to store it under some other attribute?

Reply:
As `name` attribute can be mapped only to `id` I added checking if the mapped id is unique compared to the rest of the anchors in the current document. If it's not, there is validation error thrown with a more explanatory message. Added in: https://github.com/ezsystems/ezplatform-richtext/pull/107/commits/5dabd8250ede38fa7c79eee869093dc18bf09241.

Reply:
This could be fixed this way: https://github.com/ezsystems/ezplatform-richtext/pull/107#discussion_r375926810.